smac/facade/roar_facade.py

```diff
@@ -102,4 +118,5 @@ def __init__(self,
             stats=stats,
             rng=rng,
             run_id=run_id,
+            acquisition_function_optimizer=rs,
```

Comment:
More a conceptual remark: It is a bit counterintuitive to have an optimizer called random search, but I guess there is nothing we can change here.

Reply:
I could rename it ;) and I will do so because it's actually confusing to pass random search to random search

Reply:
I tried to rename it but did not find any better name that is not extremely long or does duplicate information given in the package name.